core/App/ecp.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2011-2020 Intel Corporation. All rights reserved.
```

Comment:
change the title as 
“
Add ra protocol to setup a secure channel for API key enrollment.

This protocol is based on the key exchange lib provided by the SGX SDK.

Three new POST APIs will be added for the enrollment:
POST ehsm-addr/?Action=RA_HANDSHAKE_MSG0
POST ehsm-addr/?Action=RA_HANDSHAKE_MSG2
POST ehsm-addr/?Action=RA_GET_API_KEY

A secure channel will be established only when it passed the remote attestation, 
then the ciphertext of APP ID and API Key will be transferred from eHSM-core
enclave to the Enroll Application.
”